vscode/src/diffViewCodeActionProvider.ts

```
@@ -0,0 +1,60 @@
+/**
+ * Code action provider for diff/merge editor actions
+ * Shows "Accept Changes" and "Reject Changes" actions when viewing files
+ */
+export class DiffViewCodeActionProvider implements vscode.CodeActionProvider {
+  public static readonly providedCodeActionKinds = [vscode.CodeActionKind.QuickFix];
+
+  // Track files that were opened through VIEW_FILE action
+  private static viewedFiles = new Set<string>();
+
+  public static addViewedFile(filePath: string) {
+    this.viewedFiles.add(filePath);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)